README.adoc

```diff
@@ -242,9 +279,54 @@ initialized. With the readiness probe, you can be certain the pod will only acce
 traffic when the microservice has fully started.
 
 Similarly, navigate to `{inventory-api}/system-service` and observe that the request is successful.
+endif::[]
+
+// Cloud hosted guide instruction
+ifdef::cloud-hosted[]
+In this IBM cloud environment, you need to set up port forwarding to access the services.
+Run the following commands for the **system** service.
+```
+SYSTEM_NODEPORT=`kubectl get -o jsonpath="{.spec.ports[0].nodePort}" services system-service`
+kubectl port-forward svc/system-service $SYSTEM_NODEPORT:9080
+```
+{: codeblock}
+
+Open another command-line session by selecting **Terminal** > **New Terminal** from the menu of the IDE.
+Run the following commands to set up port forwarding to access the **inventory** service.
+```
+INVENTORY_NODEPORT=`kubectl get -o jsonpath="{.spec.ports[0].nodePort}" services inventory-service`
+kubectl port-forward svc/inventory-service $INVENTORY_NODEPORT:9080
+```
+{: codeblock}
+
+Open another command-line session by selecting **Terminal** > **New Terminal** from the menu of the IDE.
+Make a request to the system service to see the JVM system properties with the following command:
+```
+SYSTEM_NODEPORT=`kubectl get -o jsonpath="{.spec.ports[0].nodePort}" services system-service`
+curl -s http://localhost:$SYSTEM_NODEPORT/system/properties | jq
+```
+{: codeblock}
+
+The readiness probe ensures the READY state won't be `1/1`
+until the container is available to accept requests.
+Without a readiness probe, you may notice an unsuccessful response from the server.
```

Comment:
you may notice an unsuccessful response from the server.
--->
you might notice an unsuccessful response from the server.